src/test/rustdoc/empty-impl-block.rs

```diff
@@ -0,0 +1,20 @@
+#![crate_name = "foo"]
+
+// @has 'foo/struct.Foo.html'
+pub struct Foo;
+
+// @has - '//*[@class="docblock"]' 'Hello empty impl block!'
+// @has - '//*[@class="item-info"]' 'This impl block contains no items.'
+/// Hello empty impl block!
+impl Foo {}
+// We ensure that this empty block block without doc isn't rendered.
```

Comment:
Otherwise, it sounds like a rhyme ;)

```suggestion
// We ensure that this empty impl block without doc isn't rendered.
```

Reply:
:laughing: